pkg/util/mount/mount_windows.go

```diff
@@ -291,7 +291,7 @@ func lockAndCheckSubPathWithoutSymlink(volumePath, subPath string) ([]uintptr, e
 	if err != nil {
 		return []uintptr{}, fmt.Errorf("Rel(%s, %s) error: %v", volumePath, subPath, err)
 	}
-	if strings.HasPrefix(relSubPath, "..") {
+	if relSubPath == ".." || strings.HasPrefix(relSubPath, ".."+string(filepath.Separator)) {
```

Comment:
@andyzhangx, can you verify this is correct? (does filepath.Rel always produce `..\` backstep paths on windows or would a subPath of `../...` get preserved and fail this check?